File: module06/module025/module1/src/main/java/com/android/example/module06_module025_module1/ClassAAB.java
package com.android.example.module06_module025_module1;

public class ClassAAB {
	private 	com.android.example.module06_module025_module2.ClassAAC instance_var_1_0 = new com.android.example.module06_module025_module2.ClassAAC();
	private 	com.android.example.module06_module109_module1.ClassAAB instance_var_1_1 = new com.android.example.module06_module109_module1.ClassAAB();

	public void method0(
		com.android.example.module06_module109_module1.ClassAAH param0,
		com.android.example.module06_module025_module2.ClassAAA param1,
		com.android.example.module06_module109_module1.ClassAAG param2) throws Throwable {
		param0.method0("SomeString", "SomeString", "SomeString");

		param0.method2("SomeString", "SomeString", "SomeString", "SomeString");

	}

	public void method1(
		com.android.example.module06_module109_module1.ClassAAA param0,
		com.android.example.module06_module025_module2.ClassAAA param1) throws Throwable {
		java.util.Collections.emptyList().forEach( lambda0 -> {
			try {
				param0.method2("SomeString", "SomeString");

			} catch(Throwable e) { } // ignore
		});
	}
}

File: module01/module04/src/main/java/com/android/example/module01_module04/ClassAAB.kt
package com.android.example.module01_module04

@Suppress("UNUSED_PARAMETER")
class ClassAAB {
	private 	val instance_var_1_0: io.reactivex.internal.util.LinkedArrayList = io.reactivex.internal.util.LinkedArrayList(81)
	private 	val instance_var_1_1: io.reactivex.subscribers.TestSubscriber<kotlin.Any> = io.reactivex.subscribers.TestSubscriber()

	fun method0(
		param0: io.reactivex.subscribers.TestSubscriber<kotlin.Any>,
		param1: io.reactivex.internal.disposables.ListCompositeDisposable,
		param2: io.reactivex.subscribers.TestSubscriber<kotlin.Any>
	) {
		val local_var_2_3: com.android.example.module06_module321_module10.ClassAAG = com.android.example.module06_module321_module10.ClassAAG()
		local_var_2_3.method2("SomeString", "SomeString", "SomeString")

		listOf("1", "2", "3").forEach {
			listOf("1", "2", "3").forEach {
				val local_var_4_0: com.android.example.module06_module282_module1.ClassAAC = com.android.example.module06_module282_module1.ClassAAC()
				local_var_4_0.method0(com.android.example.module06_module342_module3.ClassAAG(), io.reactivex.disposables.SerialDisposable(), com.android.example.module06_module342_module3.ClassAAF())

			}
			param0.requestMore(-7934340647682723161)

		}
	}

	fun method1(
		param0: io.reactivex.subscribers.TestSubscriber<kotlin.Any>,
		param1: io.reactivex.internal.util.LinkedArrayList
	) {
	}

	fun method2(
		param0: io.reactivex.subscribers.TestSubscriber<kotlin.Any>,
		param1: io.reactivex.schedulers.TestScheduler,
		param2: io.reactivex.subscribers.TestSubscriber<kotlin.Any>,
		param3: io.reactivex.subscribers.TestSubscriber<kotlin.Any>
	) {
		param0.isCancelled()

		val local_var_2_4: com.android.example.module06_module109_module1.ClassAAG = com.android.example.module06_module109_module1.ClassAAG()
		local_var_2_4.method0("SomeString", "SomeString")

		val local_var_2_5: com.android.example.module06_module201_module1.ClassAAH = com.android.example.module06_module201_module1.ClassAAH()
		local_var_2_5.method2(io.reactivex.subscribers.TestSubscriber(), io.reactivex.subscribers.TestSubscriber(), io.reactivex.subscribers.TestSubscriber(), io.reactivex.subscribers.TestSubscriber())

		param0.onNext(kotlin.Any())

	}
}
